src/Altinn.App.Core/Models/AccessToken.cs

```diff
@@ -0,0 +1,109 @@
+using System.Text.RegularExpressions;
+
+namespace Altinn.App.Core.Models;
+
+/// <summary>
+/// Represents an OAuth 2.0 access token in JWT format
+/// </summary>
+public readonly partial struct AccessToken : IEquatable<AccessToken>
+{
+    private readonly string _accessToken;
+
+    /// <summary>
+    /// Gets the access token as a string
+    /// </summary>
+    public string Get()
```

Comment:
Could this be a `Value` property instead? Feels like that better fits the convention